src/towncrier/test/test_settings.py

```diff
@@ -194,6 +178,42 @@ def test_towncrier_toml_preferred(self):
         config = load_config(project_dir)
         self.assertEqual(config.package, "a")
 
+    def test_pyproject_name_fallback_both(self):
+        """
+        Towncrier will fallback to the [project.name] value in pyproject.toml.
```

Comment:
Just a small suggestion to help understand the scope of this test.

```suggestion
        To obtain the name of the project, when there is no dedicated [tool.towncrier] section in pyproject.toml, it will fallback to the `name` option from the [project] section.
```

Just asking. Is `towncrier` usable without a configuration section ?

Reply:
Yes. When it is a Python project with a `pyproject.toml` file, then a configuration file / setting is not required.